### PR TITLE
Code Health: Add XXX_security_bits documentation

### PR DESCRIPTION
**Pull request by InfoHunter**
This is a 'code health' commit to respond to this round of code health
Tuesday...

[skip ci]

##### Checklist

- [x] documentation is added or updated

### CODE REVIEW

File: doc/man3/EVP_SignInit.pod [hunk 1]
```diff
@@ -17,6 +18,7 @@ functions
  void EVP_SignInit(EVP_MD_CTX *ctx, const EVP_MD *type);
 
  int EVP_PKEY_size(EVP_PKEY *pkey);
+ int EVP_PKEY_security_bits(const EVP_PKEY *pkey)
```
Comment (xiaoyinl) [inline doc/man3/EVP_SignInit.pod]: Missing a semicolon here.

**Comment by InfoHunter** [comment]
Fixed, thanks!

**Comment by InfoHunter** [comment]
Add [skip ci]...

**Comment by richsalz** [comment]
squashed and merged.  Thanks!

**Pull request closed without merging**